ci/pipelines/geode-build/jinja.template.yml

```diff
@@ -222,39 +222,40 @@ jobs:
   public: true
   serial: true
   plan:
-  - get: geode-ci
-  - get: alpine-tools-image
   - aggregate:
+    - get: geode-ci
+    - get: alpine-tools-image
     - get: geode
       trigger: true
     - get: geode-build-version
       params:
         pre: ((semver-prerelease-token))
-    - do:
-      - put: concourse-metadata-resource
-      {{ init_retry()|indent(6) }}
-      - task: create_instance
-        image: alpine-tools-image
-        config:
-          platform: linux
-          params:
-            {{ common_instance_params(build_test) | indent(12) }}
-            GEODE_BRANCH: {{repository.branch}}
-            GEODE_FORK: {{repository.fork}}
-            JAVA_BUILD_VERSION: {{ java_build_version.version }}
-          run:
-            path: geode-ci/ci/scripts/create_instance.sh
-          inputs:
-          - name: attempts-log
-            path: old
-          - name: concourse-metadata-resource
-          - name: geode-ci
-          outputs:
-          - name: attempts-log
-            path: new
-          - name: instance-data
-        timeout: 15m
-        attempts: 10
+  - do:
```

Comment:
I like that this fixes the ordering issue between `init_retry` and `alpine-tools-image`. Could this `do:` be an `in_parallel:` ?

Reply:
I figured we'd do a "5.x compatibility" pass on the all the pipelines so I wanted to change as little as possible in this PR.

Reply:
Reasonable. Approving the change now.